### ISSUE TITLE
Compilation warning when installing package

### ISSUE THREAD

**Issue by felideon** [clarete/hackernews.el#3]
A compilation warning is thrown when installing the hackernews package.

On Emacs 24.3.1 installed via homebrew.

**Steps taken:**
```M-x install-package RET hackernews RET```

**What happens:**
A ```*Compile-Log*``` buffer pops up with:

```
Leaving directory `/Users/felideon/.emacs.d/elpa/hackernews-20130220.2042'

Compiling file /Users/felideon/.emacs.d/elpa/hackernews-20130220.2042/hackernews-pkg.el at Fri Oct 18 15:00:03 2013
Entering directory `/Users/felideon/.emacs.d/elpa/hackernews-20130220.2042/'

Compiling file /Users/felideon/.emacs.d/elpa/hackernews-20130220.2042/hackernews.el at Fri Oct 18 15:00:03 2013

In hackernews-retrieve:
hackernews.el:109:19:Warning: Use `with-current-buffer' rather than
    save-excursion+set-buffer

```

**Comment by shassard** [comment]
Fixed in commit 461538b7c41f3ad9960c0c113b86d0dc56cbf5e9.

**Comment by clarete** [comment]
\o/
